site/en/r1/tutorials/sequences/audio_recognition.md

```diff
@@ -402,7 +402,7 @@ change to customize the results for your own requirements.
 ### Custom Training Data
 
 By default the script will download the [Speech Commands
-dataset](https://download.tensorflow.org/data/speech_commands_v0.01.tgz), but
+dataset](https://storage.cloud.google.com/download.tensorflow.org/data/speech_commands_v0.02.tar.gz), but
```

Comment:
```suggestion
dataset](https://storage.cloud.google.com/download.tensorflow.org/data/speech_commands_v0.01.tar.gz), but
```

Let's keep it at v0.01 since this is a TF 1.x tutorial that is only in maintenance mode. What do you think?

Reply:
OK, I fix it. https://github.com/tensorflow/docs/pull/1213/commits/7e5671c73ac5d29b3d528d66ea5937c10c937998

I read audio_recognition.md and I thought Training Section's Link(Speech Commands dataset) and Advanced Training Section's Link(Speech Commands dataset) are same. 

But this doc is in maintenance mode and I also think it's better not to change the link.